js/graph_config.js

```diff
@@ -535,6 +535,66 @@ GraphConfig.load = function(config) {
                                 return getCurveForMinMaxFieldsZeroOffset(fieldName);
                         }
                         break;
+                    case 'FF_INTERPOLATED':
+                        switch (fieldName) {
+                            case 'debug[0]': // setpoint Delta
+                            case 'debug[1]': // AccelerationModified
+                            case 'debug[2]': // Acceleration
+                                return {
+                                    offset: 0,
+                                    power: 1.0,
+                                    inputRange: 1000,
+                                    outputRange: 1.0
+                                };
+                            case 'debug[3]': // Clip or Count
+                                return {
+                                    offset: -10,
+                                    power: 1.0,
+                                    inputRange: 10,
+                                    outputRange: 1.0
+                                };
+                        }
+                        break;
+                    case 'FEEDFORWARD': // replaces FF_INTERPOLATED in 4.3
+                        switch (fieldName) {
+                            case 'debug[0]': // in 4.3 is interpolated setpoint
+                                return {
+                                    offset: 0,
+                                    power: 1.0,
+                                    inputRange: maxDegreesSecond(gyroScaleMargin),
+                                    outputRange: 1.0
+                                };
+                            case 'debug[1]': // feedforward delta element
+                            case 'debug[2]': // feedforward boost element
+                                return {
+                                    offset: 0,
+                                    power: 1.0,
+                                    inputRange: 1000,
+                                    outputRange: 1.0
+                                };
+                            case 'debug[3]': // rcCommand delta
+                                return {
+                                    offset: 0,
+                                    power: 1.0,
+                                    inputRange: 10000,
+                                    outputRange: 1.0
+                                };
+                        }
+                        break;
+                    case 'FF_LIMIT':
+                        return {
+                            offset: 0,
+                            power: 1.0,
+                            inputRange: 300,
+                            outputRange: 1.0
+                        };
+                    case 'FEEDFORWARD_LIMIT':
```

Comment:
Please attach this case to line 584 to avoid duplicate code block:
```
case 'FF_LIMIT':
case 'FEEDFORWARD_LIMIT':
   return {
      ...
```

Reply:
Thanks, done.